.pipelines/nuget.yml

```diff
@@ -14,6 +14,12 @@ parameters:
   type: boolean
   default: false
 
+- name: BuildTemplate
```

Comment:
would it be simpler to just have an `IsDummyPackage` parameter at this level? the build-package-for-nuget.yml template can be called with different parameters based on it.

a yaml template file input is very flexible but requires the caller to know what kind of template file to specify.